### ISSUE TITLE
Integration Test - Price Alert

### ISSUE THREAD

**Issue by bieglich** [bitshares/bitshares-ui#2215]
**Is your feature request related to a problem? Please describe.**
Android / Chrome

Price Alert breaks the Exchange, see Screenshot. After closing by clicking "x" the Screen remains white
![screenshot_20181124-232453](https://user-images.githubusercontent.com/45090761/48973511-1a533000-f041-11e8-9007-ebf15433ed67.png)

**Comment by gibbsfromncis** [comment]
@bieglich please provide Android/Chrome versions

**Comment by gibbsfromncis** [comment]
Tested on 
**Device**: Galaxy Tab 4
**Android OS**: 4.4
**Chrome**: 68.0.xxx

Works perfect.

**Comment by sschiessl-bcp** [comment]
The versions should be the ones posted in the main integration test issue

**Comment by startailcoon** [comment]
I've had issues with the same nature on Android, where the switch account notification also breaks the page.

**Device:** Samsung Galaxy Note 8
**Android OS:** 8.0.0
**Browser:** Edge 42.0.0.2801

**Comment by gibbsfromncis** [comment]
@startailcoon Edge on Android? You're killing me :D

**Comment by sschiessl-bcp** [comment]
I think it's valid to say there is a bug, but it might only be in older version ... 

Investigation is needed if we need to change something.

**Comment by startailcoon** [comment]
@gibbsfromncis Yes, I'm used to Edge and Microsoft syncing applications from when using Windows Phone.  

There is something that breaks it, at some cases, just not sure what.

**Comment by gibbsfromncis** [comment]
@startailcoon okay I'll try to test it again

**Comment by startailcoon** [comment]
Where you able to take a look at this @gibbsfromncis ?

**Comment by sschiessl-bcp** [comment]
Ping
